setup.py

```diff
@@ -0,0 +1,25 @@
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="sars_tutorial",
+    version="0.0.1",
+    author="Massimo Quadranar",
+    author_email="max.square@gmail.com",
+    description="Sars workshop tutorial",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/mquad/sars_tutorial",
+    packages=setuptools.find_packages(),
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: MIT License",
+        "Operating System :: OS Independent",
+    ],
+    python_requires='>=3.6',
+    install_requires=[
+        'treelib',
+        'pymining'
+    ]
+)
```

Comment:
note that treelib should be locked to version `<=1.5.5`. I wonder whether deps in `environment.yml` should be part of this as well, e.g. pandas needs to be locked otherwise things break.